lib/src/args.dart

```diff
@@ -2,20 +2,24 @@ import 'package:args/args.dart';
 import 'package:meta/meta.dart';
 
 const _forceTraceLevel = 'force_trace_level';
+const _logFolder = 'log_folder';
```

Comment:
[nit] I'd prefer `directory` over `folder` in the variable names and flag name. Also I think we should use `log-directory` over `log_directory`, I made a mistake with `force_trace_level`... After this change goes in I'll fix that. https://github.com/natebosch/dart_language_server/issues/45

Reply:
SGTM; I've pushed the change :-)